File: Operacoes/bala.cpp
/*@bala: Distância entre dois pontos*/

/*@bala: Distância entre dois pontos*/

#include <iostream>
#include <cmath>
#include <iomanip>
#include <locale>
using namespace std;

int main(){
    setlocale(LC_ALL, "Portuguese");
    
    double x1, x2, y1, y2;

    cin >> x1;
    cin >> y1;
    cin >> x2;
    cin >> y2;

    double distancia = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2));

    cout << fixed << setprecision(2);
    cout << distancia << endl;
    return 0;
}

File: src/Operacoes/Bala.java
/**
 * Questão: @bala: Distância entre dois pontos
 *
 * @author UxieGu1
 */
package Operacoes;
import java.util.Scanner;
import java.lang.Math;

public class Bala {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double x1 = scanner.nextDouble();
        double y1 = scanner.nextDouble();
        double x2 = scanner.nextDouble();
        double y2 = scanner.nextDouble();

        double distancia = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));

        System.out.printf("%.2f%n", distancia);
        scanner.close();
    }
}
